### ISSUE TITLE
Проблемы русской озвучки

### ISSUE THREAD

**Issue by dosinabox** [dosinabox/g2nr_unofficial_update#53]
- [ ] Проверить:

> неправильные названия предметов в диалогах ГГ с Халвором (только субтитры);
> перевод нескольких фраз в диалогах ГГ с Беннетом в Хоринисе и Кардифом (только субтитры).

- [ ] Не озвучена реплика Hackebeil_01_00
- [ ] DIA_Oric_IAmBack_11_03 - не озвучено слово "высшим"

**Comment by dosinabox** [comment]
Неправильно переведено слово "Tor" (нужно "врата"):
DIA_Schwarzmagier_HELLO_hinterTor_15_00:

> А что скрывается за тем огромным порталом, вон там?

Неправильно переведено слово "Pforte" (нужно "проход" или "путь"):
DIA_Pyrokar_SUCCESS_11_01:

> Ты... сделал это?! Ты следовал знакам и нашел скрытый портал...

**Comment by dosinabox** [comment]
Добавить DIA_MiltenOW_Hello_Friends_15_03_add:

> Я встретил Лестера. Он в порядке.

**Comment by dosinabox** [comment]
Проверить:

> неправильные названия предметов в диалогах ГГ с Халвором (только субтитры);
> перевод нескольких фраз в диалогах ГГ с Беннетом в Хоринисе и Кардифом (только субтитры).

**Comment by Kor-Angar** [comment]
Это было в оригинале. В Готика 2 НВ было Ок

**Comment by Kor-Angar** [comment]
так.. я же играл в немецкую версию в аддоне. сейчас перепроверю :-)

**Comment by Kor-Angar** [comment]
может у меня просто не те файлы русской озвучки? ;-) надо наверное скачать какую твою версию для стим и поставить, что бы она всё исправила?

**Comment by Kor-Angar** [comment]
оказалось что всё ок
https://www.twitch.tv/videos/480228447

**Comment by dosinabox** [comment]
> оказалось что после установки всё ок

Возможно, это из-за zSpy. Он многие незаметные предупреждения превращает в заметные ошибки с выводом сообщений для дебага. А может в обновлении эта фраза исправлена. Буду изучать.

**Comment by Kor-Angar** [comment]
А если всю озвучку собрать в олин файл, она работать будет корректно?

**Comment by dosinabox** [comment]
Да.
Вопрос в том, когда появилась эта исправленная озвучка в speech3 от разработчиков/локализаторов. Если позже релиза 1.3, то мне нужно будет учитывать её в моде.

**Comment by Kor-Angar** [comment]
Я просто про то, что наверное мне проще все собрать в один файл чем собирать по кусочкам 🙂

**Comment by dosinabox** [comment]
А зачем? Я просто спрашиваю что в контенте релизной версии 1.3 с дисков Акеллы, потому что у меня их нет и не было)

**Comment by dosinabox** [comment]
> А если всю озвучку собрать в один файл, она работать будет корректно?

Да. В стим-фиксе я так и собираю, правда в два файла (оригинал и аддон).

**Comment by Kor-Angar** [comment]
> [я уже давно мог нормально спать.](https://www.youtube.com/watch?v=PYCGBwbpqgI&lc=)

должно быть "не мог"

**Comment by dosinabox** [comment]
> должно быть "не мог"

Уже было когда-то исправлено

**Comment by Kor-Angar** [comment]
Перепроверю 😀0:43, 13 февраля 2020 г., dosinabox <notifications@github.com>:
должно быть "не мог"

Уже было когда-то исправлено

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.
-- Отправлено из мобильного приложения Яндекс.Почты

**Comment by dosinabox** [comment]
А кто говорит таким же голосом, как Секоб?

**Comment by Kor-Angar** [comment]
на вскидку не скажу

**Comment by dosinabox** [comment]
DIA_Addon_PIR_6_GimmeGrog_06_00 - "костра" произносится как "костей"

**Comment by dosinabox** [comment]
В теории, этот диалог можно перенести Мике:
```cpp
AI_Output(other,self,"DIA_Addon_Hakon_MissingPeople_15_00");	//Что ты знаешь об исчезновениях людей?
AI_Output(self,other,"DIA_Addon_Hakon_MissingPeople_12_01");	//Я вижу множество людей, проходящих через восточные ворота.
AI_Output(self,other,"DIA_Addon_Hakon_MissingPeople_12_02");	//Некоторых из приходящих в город я раньше никогда не видел, а некоторые из уходящих больше не возвращаются.
AI_Output(self,other,"DIA_Addon_Hakon_MissingPeople_12_03");	//Но недавно случилось нечто действительно странное.
AI_Output(self,other,"DIA_Addon_Hakon_MissingPeople_12_04");	//В город пришел один человек... Вроде бы его звали Джо. Он что-то болтал о том, что скоро он разбогатеет.
AI_Output(self,other,"DIA_Addon_Hakon_MissingPeople_12_05");	//Он утверждал, что нашел способ пробраться в одну из городских башен, где хранится оружие ополчения.
AI_Output(self,other,"DIA_Addon_Hakon_MissingPeople_12_06");	//С тех пор я его не видел. Хотя раньше я встречался с ним каждый день.
AI_Output(self,other,"DIA_Addon_Hakon_MissingPeople_12_07");	//Я рассказал об этом ополчению.
AI_Output(self,other,"DIA_Addon_Hakon_MissingPeople_12_08");	//Но лорд Андрэ сказал, что не понимает, о чем я говорю. Он даже не знал, что это за парень.
```

Тогда не будет несоответствия голоса у Хакона)

**Comment by Kor-Angar** [comment]
на этот диалог есть отсылка у Кардифа по моему. ну вообщем есть у кого то отсылка на диалог про Джо у Хакона.

**Comment by Kor-Angar** [comment]
у Корагона

AI_Output(self,other,"DIA_Addon_Coragon_MissingPeople_09_03");	//Хакон, один из рыночных торговцев, рассказал мне очень странную историю.
AI_Output(self,other,"DIA_Addon_Coragon_MissingPeople_09_04");	//Он говорил, что каждый день он встречал одного парня, а потом тот словно исчез с лица земли. Хакон даже обратился в ополчение.

**Comment by dosinabox** [comment]
тогда нельзя

**Comment by Kor-Angar** [comment]
Герой во второй главе при первой встрече уже был магом :-) 
```D
//		AI_Output(self,other,"DIA_Oric_IAmBack_11_03");	//Я вижу, ты стал высшим магом. Мое почтение.
		AI_Output(self,other,"DIA_Oric_IAmBack_11_03");	//Я вижу, ты стал магом. Мое почтение.
```
		AI_Output (self, other, "DIA_Oric_IAmBack_11_03"); //Wie ich sehe, bist du zum Magier ernannt worden. Meine Ehrerbietung.

**Comment by dosinabox** [comment]
DIA_Marduk_Kap3_PERM_AndNow_15_02 - неправильный перевод начиная с английской версии:
> Dazu mьssten wir erst einmal wissen, wo der Dieb ist.

> For that, we would need to know first who the thief is.

> Для этого сначала нужно знать, кто этот вор.

Надо не "кто", а "где". То что Глаз украл Педро магам известно, об этом говорит Пирокар и вряд ли Мардук не в курсе:
```cpp
AI_Output(self,other,"DIA_Pyrokar_GIVEINNOSEYE_wer_11_05");	//Один из наших самых верных последователей, кандидат на священную робу Мага Огня, неожиданно изменил свою веру и сделал это в вызывающе дьявольской манере. Это Педро.
AI_Output(self,other,"DIA_Pyrokar_GIVEINNOSEYE_wer_11_06");	//Враг овладел им, нанеся тем самым ужасное поражение всем нам.
AI_Output(self,other,"DIA_Pyrokar_GIVEINNOSEYE_wer_11_07");	//Педро пробил себе дорогу мечом в наши самые священные покои и украл Глаз.
```

**Comment by dosinabox** [comment]
> Тогда не будет несоответствия голоса у Хакона)

У кого ещё есть такие несоответствия, когда реплики аддона записывали другие актеры? Нужно собрать такой список на будущее, технологии шагнули вперед и такие реплики можно создавать самостоятельно уже сейчас.

**Comment by Kor-Angar** [comment]
Где то на форуме был полный список кто кого переозвучил. 19:55, 2 марта 2023 г., dosinabox ***@***.***>:

Тогда не будет несоответствия голоса у Хакона)

У кого ещё есть такие несоответствия, когда реплики аддона записывали другие актеры?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
-- Отправлено из мобильного приложения Яндекс Почты

**Comment by dosinabox** [comment]
DIA_STRF_1107_COOK_01_01 - неправильный перевод и пропавшая шутка:

> Es gab da ein Missverständnis zwischen einem Gast, einem Hackebeil und mir.

> There was a misunderstanding between a guest, a meat cleaver, and me.

> Ну, между одним из посетителей, дровосеком, и мной произошло недоразумение.

Надо не "дровосеком", а "тесаком". К этому же тесаку отсылается фоновая реплика Hackebeil_01_00, которая в русской версии переведена неправильно и вообще не озвучена:

> (zu sich) Hackebeil...

> (to himself) Meat cleaver...

> (себе под нос) Кирка...

**Comment by dosinabox** [comment]
> У кого ещё есть такие несоответствия, когда реплики аддона записывали другие актеры? Нужно собрать такой список на будущее, технологии шагнули вперед и такие реплики можно создавать самостоятельно уже сейчас.

Переозвучка Хакона: 
[SPEECH_AI_HAKON.zip](https://github.com/dosinabox/g2nr_unofficial_update/files/13073087/SPEECH_AI_HAKON.zip)
По-моему, получилось достаточно хорошо.

**Comment by Kor-Angar** [comment]
> По-моему, получилось достаточно хорошо.

бодро

**Comment by dosinabox** [comment]
Лестер:
[SPEECH_AI_LESTER.zip](https://github.com/dosinabox/g2nr_unofficial_update/files/13180048/SPEECH_AI_LESTER.zip)
